workqueue_test.go

```diff
@@ -26,46 +27,46 @@ func TestShouldEnqueueIntoTheUnderlyingWorkQueue(t *testing.T) {
 }
 
 func TestShouldCallRegisteredAddFuncWhenAddEventIsReceived(t *testing.T) {
-	workQueue := kubehandler.NewWorkQueue("WorkqueueTest2")
+	workQueue := kubehandlerv2.NewWorkQueue("WorkqueueTest2")
 	kind := "Foo"
 	addHandlerCalled := make(chan bool, 1)
 	stopChan := make(chan struct{}, 1)
```

Comment:
remove this as well if not used

Reply:
Have removed the usage of stopCh and started using context